spec/requests/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'GET /users' do
+  it 'returns a successful response' do
+    get '/users'
+
+    expect(response).to be_successful
+    expect(response.body).to include('<h1>This is a list of all Users</h1>')
+    expect(response).to render_template(:index)
+  end
+end
+
+describe 'GET /users/:id' do
+  it 'returns a successful response' do
+    user = User.create!(name: 'Test User', post_count: 0)
+    get "/users/#{user.id}"
+
+    expect(response).to be_successful
+    expect(response.body).to include('<h1>This is a single user</h1>')
+    expect(response).to render_template(:show)
+  end
```

Comment:
- [ ] **[OPTIONAL]** Apart from writing tests that meet the expectations we also want to write tests that are descriptive. Kindly divide these tests into three 'it' blocks describing what each expectation checks for.